### ISSUE TITLE
issue with cubic chunks and multipart

### ISSUE THREAD

**Issue by Redd56** [TheCBProject/ForgeMultipart#53]
Whenever reloading a world or loading into a server and sometimes when placing them all multiparts placed will not be visible no matter what you do. it is a very big issue in my opinion as it takes using cubic chunks out of the question for this mod and project red. which in my opinion are amazing mods
edit: i just placed some down outside the spawn chunks that i couldnt see reloaded the world by going out and back into it (singleplayer) and now they apeared... im confused
it would appear that the case is that you need to reload to see ones outside of spawn chunks and that inside spawn chunks for some reason you cant reload. which is wierd
edit again: i am going to do some testing it seems to be in a patern out from spawn what happens. ill give more info once i have it
another edit: it would appear that each chunk has its own properties, either its a reload chunk or a dont reload chunk,

****reload chunk is a chunk that requires you to redownload it from the server side in order to see the microblocks****
****nonreload chunks are the exact oposite as when you redownload them the microblocks go away, either from relogging or walking back to the area****
all chunks seem to load in microbits when there is a redownloaded reload chunk nearby and seem to have much different behavor, some reload chunks become non reload and some nr become r and some stay the same etc, it is very wierd.

**Comment by Redd56** [comment]
tldr: this issue is very bad and may be a networking thing, plz fix

**Comment by covers1624** [comment]
It will require large amounts of work to achieve this. Ill be punting this to 1.14.
